packages/codemirror-graphql/package.json

```diff
@@ -14,30 +14,36 @@
   },
   "license": "MIT",
   "files": [
-    "dist",
+    "hint.esm.js",
     "hint.js",
+    "hint.js.flow",
+    "info.esm.js",
     "info.js",
+    "info.js.flow",
+    "jump.esm.js",
     "jump.js",
+    "jump.js.flow",
+    "lint.esm.js",
     "lint.js",
+    "lint.js.flow",
+    "mode.esm.js",
     "mode.js",
-    "*.flow",
+    "mode.js.flow",
     "utils",
     "variables",
-    "results",
-    "README.md",
-    "LICENSE"
+    "results"
   ],
-  "main": "dist",
```

Comment:
Do we not need a `main` for this package?

Reply:
actually we dont, it has no index! with this there, certain bundlers throw warnings and/or maybe I saw it break a webpack build once?